sycl/test-e2e/Basic/library_loading.cpp

```diff
@@ -1,16 +0,0 @@
-// REQUIRES: linux
-// RUN: %{build} -o %t.out
-// RUN: env SYCL_PI_TRACE=-1 %{run-unfiltered-devices} %t.out &> %t_trace_no_filter.txt || true
-// RUN: FileCheck --input-file=%t_trace_no_filter.txt --check-prefix=CHECK-NO-FILTER %s -dump-input=fail
-// Checks pi traces on library loading
```

Comment:
Do I understand correctly that corresponding tracing for loading of adapters is available at UR level?

Reply:
Yes, UR has equivelent tracing.